### PR TITLE
SITES-11947 - [Sites Polaris Picker][Editor] Image Component support for Polaris assets

### PR DESCRIPTION
**Pull request by LSantha**
| Q                        | A 
| ------------------------ | ---
| Fixed Issues?            | `SITES-11947` 
| Patch: Bug Fix?          |
| Minor: New Feature?      |
| Major: Breaking Change?  |
| Tests Added + Pass?      | Yes
| Documentation Provided   | Yes (code comments and or markdown)
| Any Dependency Changes?  |
| License                  | Apache License, Version 2.0

### CODE REVIEW

File: content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/editor/js/polaris-assets-picker.js [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+		console.log("imsProps: " + JSON.stringify(imsProps, null, 2));
+		PureJSSelectors.registerAssetSelectorsIms(imsProps);
+	}
+
+	function renderAssetSelectorInline() {
+		const assetSelectorProps = {
+			"discoveryURL": polarisConfiguration.discoveryUrl,
+			"repositoryId": polarisConfiguration.repositoryId,
+			"apiKey": polarisConfiguration.apiKey,
+			"orgName": "AEM Cloud Demos",
```
[the remark this answers is not in the document]
Comment (LSantha) [reply content/src/content/jcr_root/apps/core/wcm/components/image/v3/image/clientlibs/editor/js/polaris-assets-picker.js]: @icaraps we removed those clientlib changes

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java [hunk 2]
```diff
@@ -65,16 +61,31 @@ public class ImageImpl extends com.adobe.cq.wcm.core.components.internal.models.
...
+    private boolean polarisImage = false;
+
     @PostConstruct
     protected void initModel() {
+        initResource();
+        properties = resource.getValueMap();
+        String fileReference = properties.get("fileReference", String.class);
+        if (StringUtils.isNotBlank(fileReference) && fileReference.startsWith("/urn:")) {
+            polarisImage = true;
+            String repositoryId = nextGenDynamicMediaConfig.getRepositoryId();
```
[the remark this answers is not in the document]
Comment (LSantha) [reply bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java]: Thank you, I've fixed it.

File: bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java [hunk 3]
```diff
@@ -15,40 +15,37 @@
...
-import com.day.cq.dam.api.Asset;
-import com.fasterxml.jackson.annotation.JsonIgnore;
-import com.adobe.cq.wcm.core.components.internal.helper.image.AssetDeliveryHelper;
+import javax.annotation.PostConstruct;
+import javax.inject.Inject;
+import java.awt.*;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
```
Comment (bpauli) [inline bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v3/ImageImpl.java:48]: Please follow the code conventions for Java
The order of the package imports is:

java.*
_blank line_
javax.*
_blank line_
org.*
_blank line_
*
_blank line_
import static *

**Review by bpauli — APPROVED** [review]
LGTM

**Pull request merged**